src/nvim/generators/gen_api_ui_events.lua

```diff
@@ -4,14 +4,14 @@ local nvimdir = arg[1]
 package.path = nvimdir .. '/?.lua;' .. package.path
 
 assert(#arg == 7)
-local input = io.open(arg[2], 'rb')
-local proto_output = io.open(arg[3], 'wb')
-local call_output = io.open(arg[4], 'wb')
-local remote_output = io.open(arg[5], 'wb')
-local bridge_output = io.open(arg[6], 'wb')
-local metadata_output = io.open(arg[7], 'wb')
-
-local c_grammar = require('generators.c_grammar')
+input = io.open(arg[2], 'rb')
+proto_output = io.open(arg[3], 'wb')
+call_output = io.open(arg[4], 'wb')
+remote_output = io.open(arg[5], 'wb')
+metadata_output = io.open(arg[6], 'wb')
+redraw_output = io.open(arg[7], 'wb')
```

Comment:
why remove `local` ?

Reply:
That was by mistake during resolving conflicts 😅